£Lerror_reason¢DtypeEvalueXÛHTTPConnectionPool(host='test2.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x040957B0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed',))Ihttp_code¢DtypeEvalueÛQdownload_complete¢DtypeEvalueı